index.html

```diff
@@ -4381,6 +4377,75 @@ <h3>Parsing content</h3>
       [[RFC2048]] and [[RFC2046]].
     </p>
   </section>
+  <section> <h3>Things that users should be made aware of when using NFC</h3>
+    <p>
+      This section details some of the things that users ought to be aware
+      of when using NFC. It is recommendated that implementations help educate
+      the users of given facts before or when related NFC actions are performed. 
+    </p>
+    <h4>Data that is read is shared with site</h4>
+    <p>
+      When a site has access to read NFC content, then the data of the scanned
+      tags is shared with the site, in a similar way to uploading files and images.
+      The user has to trust that the site handles this data properly and in the
+      intended manner.
+    </p>
+    <h4>A site may modify and overwrite data of tags that are not made read-only</h4>
+    <p>
+      Deployed NFC solutions, like tags in stores etc, should always be made
+      read-only in order to ensure they are not modified by mistake or as part
+      of a malicious act.
+    </p>
+    <p>
+      Private tags and stickers are often unlocked (writable) from the factory
+      and the user should be aware that such tags might be overwritten/modified
+      by scanning them.
+    </p>
+    <h4>Reading a fixed (e.g. mounted) tag may expose reading location</h4>
+    <p>
+      A fixed tag may encode its ID or location in the data, meaning that reading
+      it exposes that information to the site, which then can deduct the location
+      the read took place. That combined with being logged into a service, can
+      share your location data with the site.
+    </p>
+    <p>
+      Data written is readable by other apps and sites with granted read access
+      Any NDEF data on a tag can be read by any app or web site with the proper
+      access, so if that is not intended then the data should be encrypted in a
+      secure manner that only who is supposed to read it can.
+    </p>
+    <h4>Multiple tags may be within the reading field at the same time</h4>
+    <p>
+      NFC can only read one tag at the time, but multiple tags can be detected
+      and one of the tags can be selected as the tag to communicate with.
+    </p>
+    <p> 
+      Use cases for this could be having multiple smart cards (NFC based) in
+      your wallet and not wanting to take the card out.
+    </p>
+    <p>
+      This is mostly useful for payment cards and travel cards that are read
+      by external hardware and thus not a use-case for Web NFC. For Web NFC,
+      we do not allow reading when there are multiple tags available.
+    </p>
+    <p>
+      There is an attack vector, where someone places another malicious NFC
+      tag/sticker on top of a legitimate tag, in order to load the wrong
+      app/site, or inject wrong data into the right app/site.
+    </p>
```

Comment:
The sites should be prepared to read _any_ tags - there is no such thing as "reading an expected tag". So I am not sure if this threat is real.

Reply:
sure there is... you can clone a tag and change the url by adding some russian latin look alike chars in the url instead... or just change the data

Reply:
So the fraud is with the tags themselves. Could we filter out / warn about this kind of attack?
Is there a standard way to vet URLs?

Reply:
The fraud is that you add a cloned and modified tag on top of a real one. People have done this with QR codes as well. It might take you to a whole different url and some users just don't notice. We don't allow reading if there are more tags close by so we can avoid this issue pretty much

Reply:
We can avoid the overlapping, but I wonder could we also warn about _possible_ fake URLs even when reading single tags? It is a well known attack, so there might be some standard solution to that already.